### PR TITLE
Fix YouTube channels contination

### PR DESCRIPTION
**Pull request by TiA4f8R**
- [x] I carefully read the [contribution guidelines](https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md) and agree to them.
- [x] I have tested the API against [NewPipe](https://github.com/TeamNewPipe/NewPipe).

Fix YouTube channels continuation extraction by using the same method as @XiangRongLin in #567.

APK to test: https://github.com/TiA4f8R/NewPipe/suites/2178388837/artifacts/44814609

(Sorry for the commits, I can't use my computer right now.)

### CODE REVIEW

**Review by pooppp3 — APPROVED** [review]
hi

**Review by FireMasterK — APPROVED** [review]

File: extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeChannelExtractor.java [hunk 1]
```diff
@@ -254,27 +264,45 @@ public boolean isVerified() throws ParsingException {
...
 
-    private Page getNextPageFrom(final JsonObject continuations) {
+    private Page getNextPageFrom(final JsonObject continuations) throws IOException, ExtractionException {
         if (isNullOrEmpty(continuations)) {
             return null;
         }
 
         final JsonObject continuationEndpoint = continuations.getObject("continuationEndpoint");
         final String continuation = continuationEndpoint.getObject("continuationCommand").getString("token");
         final String clickTrackingParams = continuationEndpoint.getString("clickTrackingParams");
```
Comment (B0pol) [inline extractor/src/main/java/org/schabi/newpipe/extractor/services/youtube/extractors/YoutubeChannelExtractor.java:303]:
```suggestion
```
it should be removed if not used

**Review by B0pol — CHANGES_REQUESTED** [review]

**Comment by XiangRongLin** [comment]
The mocks for the other tests also need to be regenerated

**Comment by TobiGr** [comment]
@XiangRongLin Can you check, whether I've done this right?

**Review by B0pol — APPROVED** [review]

**Comment by XiangRongLin** [comment]
> @XiangRongLin Can you check, whether I've done this correctly?

From a glance it looks good.

Basically if the tests pass with the mock downloader then its good.

**Pull request merged**